.eslintrc.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
Not blocker: as I see, you use commits convention, but in a little bit strange form. https://www.npmjs.com/package/@commitlint/config-conventional - maybe this is what you are looking for